contracts/MOROFT.sol

```diff
@@ -13,6 +13,8 @@ contract MOROFT is IMOROFT, OFT {
         address _delegate,
```

Comment:
Should follow our style guide and rename all variables in this file, as well as in an interface.